sale_coupon_incompatibility/models/sale_coupon_program.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import _, fields, models
+
+
+class SaleCouponProgram(models.Model):
+    _inherit = "sale.coupon.program"
+
+    incompatible_promotion_ids = fields.Many2many(
+        comodel_name="sale.coupon.program",
+        relation="sale_coupon_program_incompatibility_rel",
+        column1="program_id",
+        column2="incompatible_program_id",
+        compute="_compute_incompatible_promotion_ids",
+        inverse="_inverse_incompatible_promotion_ids",
+        readonly=False,
+        store=True,
+        string="Incompatible Promotions",
+    )
+
+    def _compute_incompatible_promotion_ids(self):
```

Comment:
No depends is needed here?

Reply:
Can you answer about this?

Reply:
Yes, checking it twice, the compute isn't necessary at all. It will work with just the inverse